### PR TITLE
Add Screencast.app v0.0.4

### PR DESCRIPTION
**Pull request by sanyer**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/screencast.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'screencast' do
+  version '0.0.4'
+  sha256 'ce662e83464e94495f782e5656d3a541b7dd4095eb400622e0e7e0cc04ecae7d'
+
+  url "https://github.com/soh335/Screencast/releases/download/#{version}/Screencast.dmg"
+  appcast 'https://raw.githubusercontent.com/soh335/Screencast/master/appcast.xml',
```
Comment (vitorgalvao) [inline Casks/screencast.rb]: Use the releases URL instead (https://github.com/soh335/Screencast/releases.atom).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/screencast.rb (hunk 1 above)
Comment (sanyer) [inline Casks/screencast.rb]: Even if `find_sparkle_appcast` founds something?

Comment (vitorgalvao) [inline Casks/screencast.rb]: > Even if `find_sparkle_appcast` founds something?

Yes. Sparkle appcasts break and are abandoned frequently. Those are built automatically and are better for our purposes.

Comment (sanyer) [inline Casks/screencast.rb]: Changed

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**